.travis.yml

```diff
@@ -39,8 +39,7 @@ script:
 - npm install
 - dfx start --background
 - sleep 5
-- make
-- make install
-- nohup make run &
```

Comment:
Where are we starting the bootstrap server? `cd canisters/index/assets && python -m SimpleHTTPServer 8000` ?

Reply:
Also recall that we cannot use SimpleHTTPServer here because we MUST go over HTTPS or the user library won't work.